build/BUILD

```diff
@@ -67,27 +67,57 @@ DOCKERIZED_BINARIES = {
     },
 }
 
-[container_image(
-    name = binary + "-internal",
+[[container_image(
```

Comment:
Are you not using a for loop here because that would require you to use a .bzl file? I kind of feel like that's cheating

Reply:
correct, you can only define macros in `.bzl` files.

(I wrote this pattern enough here that maybe I should refactor it...)